extension/webviews/process-editor/src/start/action.ts

```diff
@@ -22,12 +23,14 @@ export namespace StartProcessAction {
   }
 }
 
+const StartProcessRequest: RequestType<string, void> = { method: 'startProcess' };
+
 @injectable()
 export class StartProcessActionHandler implements IActionHandler {
-  constructor(@inject(VsCodeApi) private vscodeapi: VsCodeApi) {}
+  constructor(@inject(Messenger) private messenger: Messenger) {}
 
   handle(action: StartProcessAction) {
-    this.vscodeapi.postMessage({ command: 'startProcess', processStartUri: action.processStartUri });
+    this.messenger.sendRequest(StartProcessRequest, HOST_EXTENSION, action.processStartUri);
```

Comment:
I think this could also be a normal glsp action which then is catched in the node backend if needed. But maybe we need to move the server action handler to the normal server for that.